staging/src/k8s.io/apiserver/pkg/storage/testing/utils.go

```diff
@@ -340,3 +343,67 @@ func (s sortablePodList) Less(i, j int) bool {
 func (s sortablePodList) Swap(i, j int) {
 	s[i], s[j] = s[j], s[i]
 }
+
+func verifyEvents(t *testing.T, w watch.Interface, events []watch.Event, strictOrder bool) {
```

Comment:
We should refactor in a way that we reuse testCheckResult (either by using it here or by testCheckResult using the code of this function).

Please also move to a separate PR (might be the same with other tests).